R/global_sessions_config.R

```diff
@@ -56,13 +58,18 @@ global_sessions_config <- function(
   sign_in_providers = "email",
   is_email_verification_required = TRUE,
   is_auth_required = TRUE,
-  sentry_dsn = NULL
+  sentry_dsn = NULL,
+  cookie_expiration = 365L
```

Comment:
can we call it "cookie_expires" to match up better with the option in js-cookie

Reply:
e331a9d73c1be00ac5890acf54976c81ced54bd5